### PR TITLE
Dit 2313

### PR DESCRIPTION
**Pull request by warrenskiwork1979**

### CODE REVIEW

File: test/unit/views/CommodityCodeDigitsViewSpec.scala [hunk 1]
```diff
@@ -28,12 +28,14 @@ class CommodityCodeDigitsViewSpec extends StringViewBehaviours {
 
   val form = new CommodityCodeDigitsFormProvider()()
 
-  def createView = () => commodityCodeDigits(frontendAppConfig, form, NormalMode)(fakeRequest, messages)
+  val goodsName = "goods"
```
Comment (DavidGregory084) [inline test/unit/views/CommodityCodeDigitsViewSpec.scala:31]: I think this should have a different name from the default so we can verify we aren't getting the default

File: test/unit/controllers/CommodityCodeDigitsControllerSpec.scala [hunk 2]
```diff
@@ -35,6 +35,7 @@ class CommodityCodeDigitsControllerSpec extends ControllerSpecBase {
 
   val formProvider = new CommodityCodeDigitsFormProvider()
   val form: Form[String] = formProvider()
+  val goodsName = "goods"
```
Comment (DavidGregory084) [inline test/unit/controllers/CommodityCodeDigitsControllerSpec.scala:38]: Same here

**Review by DavidGregory084 — APPROVED** [review]
Looks good 👍

File: test/unit/controllers/CommodityCodeDigitsControllerSpec.scala (hunk 2 above)
Comment (warrenskiwork1979) [reply test/unit/controllers/CommodityCodeDigitsControllerSpec.scala]: Modified in new commit

File: conf/messages.en [hunk 3]
```diff
@@ -287,9 +287,11 @@ commodityCodeBestMatch.checkYourAnswersLabel = Have you found a commodity code?
 commodityCodeBestMatch.error.required = Select yes if you have found a commodity code
 
 commodityCodeDigits.title = What is the commodity code you found for your item? - Manage your Advance Tariff Rulings - GOV.UK
```
Comment (DavidGregory084) [inline conf/messages.en:289]: Sorry missed this first time - is the title supposed to be updated too?

**Pull request merged**